### PR TITLE
Add reducers and actions

### PR DESCRIPTION
**Pull request by wayungi**
### This PR adds ;
- add reducers
- actions
the books and categories

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/** @format */
+
+const ADD_BOOK = 'add_book';
+const REMOVE_BOOK = 'remove_book';
```
Comment (t-yanick) [inline src/redux/books/books.js]: - [x] As it is a requirement for this project, " `Write your book's actions and reducer (in /src/redux/books/books.js) using the ducks pattern.` ", please kindly correct your actions to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Something like so: 
`const ADD_BOOK = bookstore/books/ADD_BOOK`,
`const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/** @format */
+
+const CHECK_STATUS = 'check_status';
```
Comment (t-yanick) [inline src/redux/categories/categories.js]: - [x] As it is a requirement for this project, " `Write your categories actions and reducer (in /src/redux/categories/categories.js) using the ducks pattern.` ", please kindly correct your actions to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Something like so: 
`CHECK_STATUS = bookstore/categories/CHECK_STATUS`

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @wayungi, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean and organized code

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js (hunk 2 above)
Comment (wayungi) [reply src/redux/categories/categories.js]: All requested changes have been resolved @t-yanick

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @wayungi 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**